### DISCUSSION TITLE
Why is the running time of other wind directions greater than that of 270?

### DISCUSSION THREAD

**Discussion by winder-q** [NatLabRockies/floris#280] [Q&A] [answered]
hello，
In the wake calculation of the wind farm(I take the contents of the "getting started" folder as an example, and replace the parameters with the data of 80 wind turbines in the horns_rev_1 wind farm), why is the running time in other wind directions much longer than the time in the 270-degree wind direction? 
In addition, even without "fi.calculate_wake()", can it run normally and get the result?

**Comment by bayc** [comment, marked as answer]
It is possible that, depending on the wind direction, that some calculations can take longer as there will be additional turbines included in certain calculations, but I wouldn't initially expect this to be a noticeable difference. Can you provide more details on the wind directions used and the difference in timings?

And `fi.calculate_wake()` is the call for FLORIS to solve the wakes, which is needed to get the results (turbine powers, etc.). Can you further explain what you mean by "run normally and get the result"?
